deployment/src/main/java/io/quarkiverse/poi/deployment/POIProcessor.java

```diff
@@ -32,6 +32,7 @@ NativeImageFeatureBuildItem nativeImageFeature() {
     void indexTransitiveDependencies(BuildProducer<IndexDependencyBuildItem> index) {
         index.produce(new IndexDependencyBuildItem("org.apache.xmlbeans", "xmlbeans"));
         index.produce(new IndexDependencyBuildItem("org.apache.poi", "poi-ooxml-full"));
+        index.produce(new IndexDependencyBuildItem("org.apache.poi", "poi-scratchpad"));
```

Comment:
Depending on the size, I wonder if it should be a different extension.

Reply:
I was thinking if it would be possible to make support for different formats configurable with build time configuration? I believe quite some Quarkus features can be configured in such way... 
For example scratchpad could be conditionally included? Or maybe even something more fine-grained, if this makes sense at all...

Reply:
I don't think that's possible without a separate extension